package.json

```diff
@@ -443,6 +443,11 @@
                     "when": "view == objectExplorer && !config.mssql.useLegacyConnectionExperience && viewItem =~ /\\btype=(disconnectedServer|Server)\\b/",
                     "group": "1_MSSQL_serverAndDbManagement@2"
                 },
+                {
+                    "command": "mssql.setAsWorkspaceDefault",
```

Comment:
I'm not sure this command is common enough to justify adding it to the Server node's context menu in Object Explorer. Since there are already quite a few server/database commands, adding more could clutter the menu. An alternative UX might be to expose this via the command palette, using a quick pick menu to select the server.